client/v3/README.md

```diff
@@ -89,4 +89,4 @@ Client request size limit is configurable via `clientv3.Config.MaxCallSendMsgSiz
 
 ## Examples
 
-More code examples can be found at [GoDoc](https://godoc.org/go.etcd.io/etcd/clientv3).
+More code examples can be found at [GoDoc](https://godoc.org/github.com/rpcxio/etcd/clientv3).
```

Comment:
The working path is: 

https://pkg.go.dev/go.etcd.io/etcd/client/v3

If you really want to point at examples, I would recommend a direct link to: 

https://github.com/etcd-io/etcd/tree/master/tests/integration/clientv3/examples